resources/public/api/conf/3.0/schemas/create_request.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Brought Forward Loss",
+  "description": "Brought Forward Loss",
+  "type": "object",
+  "properties": {
+    "taxYearBroughtForwardFrom": {
+      "type": "string",
+      "description": "The tax year to which the loss applies. The start year and end year must not span two tax years. No gaps are allowed, for example, 2020-22 is not valid. The minimum tax year is 2018-19. The tax year must have ended.",
+      "example": "2020-21"
+    },
+    "typeOfLoss": {
+      "type": "string",
+      "enum": [
+        "self-employment",
+        "self-employment-class4",
+        "uk-property-fhl",
+        "uk-property-non-fhl",
+        "foreign-property-fhl-eea",
+        "foreign-property"
+      ],
+      "description": "The type of business the loss applies to."
+    },
+    "businessId": {
+      "type": "string",
+      "description": "An identifier for the business, unique to the customer. Must conform to the regular expression",
+      "example": "XAIS12345678910",
+      "pattern": "^X[A-Z0-9]{1}IS[0-9]{11}$"
+    },
+    "lossAmount": {
+      "type": "number",
+      "description": "Brought Forward Loss amount related to the type of loss, specified. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
```

Should there be a comma here?
```suggestion
      "description": "Brought Forward Loss amount related to the type of loss specified. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
```